test/arithmetic/floating_point/floating_point_adder_singlepath_test.dart

```diff
@@ -871,7 +871,7 @@ void main() {
     test(
         'FP: simple adder with mixed explicit/implicit j-bit IO '
         'widening exhaustive', () {
-      for (final outMantissaWidth in [3, 4, 5, 6, 7, 8, 9, 15]) {
+      for (final outMantissaWidth in [3, 4, 5, 6, 7, 8, 9]) {
```

Comment:
just curious: why remove 15?

Reply:
Tests are taking a long time and 9 is wide enough to cross a significant threshold (mantissa + jbit) * 2 = 8.